src/components/Fonts.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react"
+import { Global } from "@emotion/react"
+
+const Fonts = () => (
+  <Global
+    styles={`
+      /* latin */
+      @font-face {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 300 800;
+        font-display: swap;
+        src: url('/fonts/Inter.ttf') format('truetype');
```

Comment:
Wondering is we should also add `woff`/`woff2` formats for modern browsers. cc @nloureiro

Reply:
@TylerAPfledderer yes! There is a file available in the author's repo I linked that I didn't see previously.